src/polyphy/core/continuous2D.py

```diff
@@ -57,7 +58,7 @@ def __load_from_file__(self):
         self.data = ti.tools.image.imread(self.ROOT + self.input_file).astype(PPTypes.FLOAT_CPU) / 255.0
         self.DATA_RESOLUTION = self.data.shape[0:2]
         self.N_AGENTS = PPConfig.N_AGENTS_DEFAULT
-        self.DOMAIN_SIZE = (PPConfig.DOMAIN_SIZE_DEFAULT, PPConfig.DOMAIN_SIZE_DEFAULT \
```

Comment:
@pjflux2001 Please double check this. The backslash is there to continue the formula past the line break. This type of autoformatting change already caused issues before and actually silently broke the code, so just need to be sure this isn't the case here.

Reply:
@OskarElek Thanks for pointing it out. Resolved.